components/omega/src/ocn/HorzMesh.cpp

```diff
@@ -559,6 +565,51 @@ void HorzMesh::computeEdgeSign() {
    EdgeSignOnVertexH = createHostMirrorCopy(EdgeSignOnVertex);
 } // end computeEdgeSign
 
+//------------------------------------------------------------------------------
+// set computational masks for mesh elements
+// TODO: this is just a placeholder, implement actual masks for edges, cells,
+// and vertices
+void HorzMesh::setMasks(int NVertLevels) {
+
+   EdgeMask = Array2DR8("EdgeMask", NEdgesSize, NVertLevels);
+
+   OMEGA_SCOPE(o_EdgeMask, EdgeMask);
+
+   parallelFor(
+       {NEdgesAll}, KOKKOS_LAMBDA(int Edge) {
+          for (int K = 0; K < NVertLevels; ++K) {
+             o_EdgeMask(Edge, K) = 1.0;
```

Comment:
These o_ arrays should follow the naming conventions using a CamelCase name starting with a capital letter.